assets/scss/app.scss

```diff
@@ -1,7 +1,8 @@
 @import '../external/Font-Awesome/scss/font-awesome';
-@import '../external/jquery-ui-autocomplete/jquery-ui';
-@import '../external/jquery-ui-autocomplete/jquery-ui.structure';
-@import '../external/jquery-ui-autocomplete/jquery-ui.theme';
+@import '../external/jquery-ui-autocomplete/jquery-ui.css';
+@import '../external/jquery-ui-autocomplete/jquery-ui.structure.css';
+@import '../external/jquery-ui-autocomplete/jquery-ui.theme.css';
+
```

Comment:
I had to change the imports of the css files since the build with the sass command was breaking with out the '.css' endings of the files.